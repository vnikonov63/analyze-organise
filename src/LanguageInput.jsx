export default function LanguageInput({
  flag,
  country,
  setCurrentWord,
  currentWord,
}) {
  return (
    <>
      <p>{flag}</p>
      <input
        type="text"
        placeholder=""
        onChange={(e) => setCurrentWord(e.target.value)}
        value={currentWord}
      />
    </>
  );
}
