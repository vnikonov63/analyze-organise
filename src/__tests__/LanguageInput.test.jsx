import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import LanguageInput from "../LanguageInput";

test("snapshot testing with nothing in cart", () => {
  const { asFragment } = render(<LanguageInput />);
  expect(asFragment()).toMatchSnapshot();
});
