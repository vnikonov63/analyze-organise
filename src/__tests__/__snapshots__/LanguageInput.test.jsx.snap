// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot testing with nothing in cart 1`] = `
<DocumentFragment>
  <p />
  <input
    placeholder=""
    type="text"
  />
  <p>
    ‚ùå
  </p>
</DocumentFragment>
`;
